src/build/onboarding/ui/app.tsx

```
@@ -36,6 +40,36 @@ interface AppProps {
+async function runRunnerCommand(runner: string, args: string[]): Promise<{ success: boolean, output: string[] }> {
+  const { command, args: runnerArgs } = splitRunnerCommand(runner)
+
+  return new Promise((resolve) => {
+    const child = spawn(command, [...runnerArgs, ...args], {
+      stdio: ['ignore', 'pipe', 'pipe'],
+    })
+    const output: string[] = []
+
+    const append = (chunk: Buffer | string) => {
+      const text = typeof chunk === 'string' ? chunk : chunk.toString('utf8')
+      for (const rawLine of text.split(/\r?\n/)) {
```

ESLint

[error] Move this regular expression to module scope to avoid re-compilation on every call.

(e18e/prefer-static-regex)